gelf/tcpwriter_test.go

```diff
@@ -152,7 +152,7 @@ func TestExtraDataTCP(t *testing.T) {
 		Host:     "fake-host",
 		Short:    string(short),
 		Full:     string(full),
-		TimeUnix: float64(time.Now().Unix()),
+		TimeUnix: float64(time.Now().Unix()) / 1e9,
```

Comment:
Please add UnixNano for all tests.

Reply:
I think this should be corrected now - I believe I simply forgot to add "Nano" in the tests....